packages/bot/src/components/transactions.jsx

```diff
@@ -88,20 +88,23 @@ const Transaction = ({ contract }) => {
                         </div>
                     </td>
                     <td className='transactions__col'>
-                        <div
-                            className={className(
-                                'transactions__inline',
-                                'transactions__middle',
-                                [Math.sign(contract.profit) !== -1 ?
-                                    'transactions__green' :
-                                    'transactions__red'])}
-                        >
-                            <Money
-                                amount={Math.abs(contract.profit)}
-                                currency={contract.currency}
-                            />
-                        </div>
-
+                        {contract.profit ?
+                            <div
+                                className={className(
+                                    'transactions__inline',
+                                    'transactions__middle',
+                                    [Math.sign(contract.profit) !== -1 ?
+                                        'transactions__green' :
+                                        'transactions__red'])}
+                            >
+                                <Money
+                                    amount={Math.abs(contract.profit)}
+                                    currency={contract.currency}
+                                />
+                            </div>
+                            :
+                            <span />
```

Comment:
We can return `<React.Fragment />` here or even `null`, or do we explicitly need this?

Reply:
@aaron-binary yes cause styles is now, percentage for tds so if i remove it, the last icon will take the space and break the styles.